docs/logging/user_guide.rst

```diff
@@ -261,7 +261,7 @@ checks and consumes one log message from the list of recorded log calls to
 check a sequence of log messages.
 
 The ``mock`` procedures ensures that all future calls to the logger
```

Comment:
BTW, could you fix 'procedures ensures' to either 'procedure ensures' or 'procedures ensure'?